client/blocks/reader-full-post/index.jsx

```diff
@@ -451,6 +451,7 @@ export class FullPostView extends React.Component {
 										commentCount={ commentCount }
 										maxDepth={ 1 }
 										commentsFilterDisplay={ COMMENTS_FILTER_ALL }
+										showConversationFollowButton={ config.isEnabled( 'reader/conversations' ) }
```

Comment:
since we have `shouldShowConversationFollowButton` incorporating the `config` value for reader/conversations, I don't think we need this as a prop as well

Reply:
Ah, but we do :) Even after the feature ships, there'll be some places in Calypso that render the `Comment` block (like Comments Management) that don't want the conversation follow button.

Reply:
As I noted above, we do still need the prop, but I could just set this to `true` since the config flag is being checked in `shouldShowConversationFollowButton`. Good shout 👍